### ISSUE TITLE
AttributeError: 'Router' object has no attribute 'tranport'

### ISSUE THREAD

**Issue by fumihiko-takahashi** [MKuranowski/pyroutelib3#5]
Sorry. I am getting the following error yet. This is a typo.
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ubuntu/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py", line 401, in findNode
    self.getArea(lat, lon)
  File "/home/ubuntu/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py", line 220, in getArea
    self.loadOsm(filename)
  File "/home/ubuntu/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py", line 267, in loadOsm
    self.storeWay(wayId, wayData["tag"], wayNodes)
  File "/home/ubuntu/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py", line 377, in storeWay
    if oneway not in ["-1", "reverse"] or self.tranport == "foot":
  File "/home/ubuntu/.pyenv/versions/3.6.6/lib/python3.6/site-packages/pyroutelib3.py", line 424, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'Router' object has no attribute 'tranport'
```

**Comment by MKuranowski** [comment]
Fixed in 1.0post2. Any more typos you have found :p?

**Comment by fumihiko-takahashi** [comment]
thx! no problem!
